modules/web-cli/assets/js/core/backwards-compatibility/commands.js

```diff
@@ -13,7 +12,7 @@ export default class CommandsBackwardsCompatibility extends Module {
 			// Regex takes the first letter and convert it to lower case.
 			componentName = componentName.replace( /^./, ( val ) => val.toLowerCase() );
 
-			Helpers.softDeprecated(
+			elementor.devTools.deprecation.deprecated(
```

Comment:
The `web-cli` module should not be depended on other modules.
Please exclude it from this PR

Reply:
Do you want to duplicate the methods?

Reply:
yes

Reply:
The `web-cli` both PHP, and JS currently does depend on core, so I did the same, `web-cli` uses `dev-tools` module, Think we need to create another task, make `web-cli` by independent .